pkg/router/route.go

```diff
@@ -114,27 +107,36 @@ func (rt *Route) findNode(fullPath string) (*Node, bool) {
 	lowerPath := strings.ToLower(fullPath)
 	var n interface{}
 	var found bool
-	if n, found = rt.searchWildcard(lowerPath); !found {
-		rt.lock.RLock()
-		defer rt.lock.RUnlock()
-		if n, found = rt.tree.Get(lowerPath); !found {
-			return nil, false
-		}
+	rt.lock.RLock()
+	defer rt.lock.RUnlock()
+	tireNode, _, _ := rt.tree.Get(lowerPath)
+	found = tireNode != nil
+	if !found {
+		return nil, false
+	}
+	n = tireNode.GetBizInfo()
+	if n == nil {
+		return nil, false
 	}
 	return n.(*Node), found
 }
 
-func (rt *Route) searchWildcard(fullPath string) (*Node, bool) {
+func (rt *Route) matchNode(fullPath string) (*Node, bool) {
```

Comment:
it seems this function is the same as findNode?

Reply:
find Node only traversing the tree, ignore  path variable  :=var . 
'/test/test/xxx'  can match '/test/test/:a'  , but '/test/test/xxx' can't find '/test/test/:a'